### PR TITLE
Feature/review me

### PR DESCRIPTION
**Pull request by CastelloGovender**
Make 1 comment on this PR each, non-duplicates, please

### CODE REVIEW

**Review by RPoolman — APPROVED** [review]
Seems legit, a weird decision to use a case statement for a calculator app, might use in future and test it out.

**Review by RPoolman — APPROVED** [review]

**Comment by RPoolman** [comment]
Seems like the app is well underway with all the applicable features working.

File: README.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 Lets simulate some merge conflicts
 
 # Step 3
-Changed to step 3 on master
\ No newline at end of file
+Changed to step 3 on master
```
Comment (kmgwenya) [inline README.md]: weak sauce

Comment (Will-A999) [inline README.md:5]: Awesome stuff.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CastelloGovender/MergeConflicts.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/CastelloGovender/MergeConflicts/issues"
```
Comment (ghost) [inline package.json:17]: GIT IS AWESOME BUT ANNOYINH

File: README.md [hunk 3]
```diff
@@ -2,4 +2,7 @@
...
+
```
Comment (GMahomed) [inline README.md:6]: Random comment

File: README.md (hunk 1 above)
Comment (linkie-thelele) [inline README.md:5]: Great stuff!

Comment (FwilsonRetro) [inline README.md:5]: Thanks for the workshop

File: package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "main": "server.js",
+  "private": "true",
+  "scripts": {
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CastelloGovender/MergeConflicts.git"
+  },
+  "author": "",
```
Comment (RPoolman) [inline package.json:14]: Maybe specify the author and in the case of no author, set a default value like "None"

**Review by willemLieb — APPROVED** [review]

**Review by FwilsonRetro — APPROVED** [review]

**Pull request closed without merging**